Documentation/configuration.md

```diff
@@ -1,34 +1,36 @@
-## Configuration
+# Configuration
```

Comment:
by way of explanation, @zbwright, etcd clients like flannel can use the etcd v3 _or_ v2 apis when communicating with an etcd v3 service. So and etcd v3 binary can serve either the v2 http api and the v3 grpc api to a client. The two apis cannot be used interchangeably and they have separate data stores.

(No change required for this comment)